filament/backend/include/backend/DriverEnums.h

```diff
@@ -938,24 +916,42 @@ enum class SamplerCompareFunc : uint8_t {
     N           //!< Never. The depth / stencil test always fails.
 };
 
+enum class SamplerYcbcrModelConversion : uint8_t {
+    RGB_IDENTITY = 0,
+    YCBCR_IDENTITY=1,
+    YCBCR_709=2,
+    YCBCR_601=3,
+    YCBCR_2020=4,
```

Comment:
more consistent formatting here pls

for example
`YCBCR_IDENTITY = 1,` instead